src/bluesearch/entrypoint/database/convert_pdf.py

```diff
@@ -63,22 +67,38 @@ def init_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         help="The port of the GROBID server.",
     )
     parser.add_argument(
-        "input_pdf_path",
+        "input_path",
         type=pathlib.Path,
-        metavar="INPUT-PDF-PATH",
-        help="The path of the input PDF file.",
+        metavar="INPUT-PATH",
+        help="The path to a single PDF file or a directory with many PDF files.",
```

Comment:
Maybe this should be clarified that the 'globing' is recursive (line 163).

Reply:
Good catch, chaned in fc3a7c0

Reply:
Good catch! Changed in fc3a7c0

Reply:
Great!